open-banking-brasil-certificate-standards-1_ID1.md

```diff
@@ -343,4 +346,4 @@ otherName.3 = 2.16.76.1.3.7;UTF8:<INSS Number>
 | 2 | Direitos   Creditórios | /invoice-financings/* | ICP WEB SSL | Required |
 | 3 | Pagamentos | /payments/* | ICP WEB SSL | Required |
 
-_(*)_ if for some reason this endpoint should be public, which is not recommended.
+_(**)_ if for some reason this endpoint should be public, which is not recommended.
```

Comment:
Sugestão de tradução: "if publishing this endpoint is required, which is not mandatory."

Reply:
removido